rust/osm2lanes/src/transform/lanes_to_tags/mod.rs

```diff
@@ -56,6 +56,12 @@ impl Lane {
 pub fn lanes_to_tags(lanes: &[Lane], locale: &Locale, config: &Config) -> TagsResult {
     let mut tags = Tags::default();
     let mut oneway = false;
+
+    if !lanes.iter().any(|lane| lane.is_motor() || lane.is_bus()) {
+        tags.checked_insert("highway", "path")?;
```

Comment:
What if this method took `&Road` and we just used `Highway` here? This seems super brittle; what if it was `steps` or some other kind of footway?

Reply:
yeah, this will need to change, will refactor as part of a future PR :)